.gitignore

```diff
@@ -36,3 +36,5 @@ nohup.out
 multirun
 DC2_*.out
 input_images/
+case_studies/strong_lensing/output_yaml
+case_studies/strong_lensing/galsim_run.log
```

Comment:
Better to create a `.gitignore` within `case_studies/strong_lensing` rather than modifying the top-level one.

Reply:
Sounds good, just added a new one in the latest commit.